doc/api/n-api.md

```diff
@@ -4845,6 +4912,7 @@ This API may only be called from the main thread.
 [Section 6.1.6]: https://tc39.github.io/ecma262/#sec-ecmascript-language-types-number-type
 [Section 6.1.7.1]: https://tc39.github.io/ecma262/#table-2
 [Section 9.1.6]: https://tc39.github.io/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots-defineownproperty-p-desc
+[Section 8.7]: https://tc39.es/ecma262/#sec-agents
```

Comment:
Nit: Should be before `[Section 9.1.6]` alphabetically?

Reply:
Done.